sdk/tests/conformance/programs/program-test.html

```diff
@@ -331,10 +331,6 @@
     assertMsg(gl.getProgramParameter(progGood2, gl.LINK_STATUS) == false,
               "linking should fail with in-use formerly good program, with new bad shader attached");
 
-    // Invalid link leaves previous valid program intact.
```

Comment:
I suppose we can remove this in favor of the more detailed tests, but I generally don't like removing tests like this.

Reply:
So you prefer to leave the old test unchanged (restore the deleted section)? (Just to double check I understand correctly)

Reply:
I prefer leaving the old test, but I don't mind strongly.

Reply:
I have one more reason to delete this as current chromium implementation didn't handle this properly. Separate this part of testing into the new program-handling.html would make suppression a bit easier for us. So I think I will keep it removed.

Reply:
I don't like this sort of gamesmanship. Please focus on fixing the failing subtests, and don't focus on overall pass rate.

Reply:
FYI I'm working on fixing the handling issue at the same time.

Reply:
Restored.